apps/web/components/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -55,12 +55,14 @@ function getNavigation(props: {
   installedAppsNumber: number;
 }) {
   const { eventType, t, enabledAppsNumber, installedAppsNumber, enabledWorkflowsNumber } = props;
+  const multipleDuration = eventType.metadata?.multipleDuration?.map((duration) => ` ${duration}`);
+
   return [
     {
       name: "event_setup_tab_title",
       href: `/event-types/${eventType.id}?tabName=setup`,
       icon: Icon.FiLink,
-      info: `${eventType.length} Mins`, // TODO: Get this from props
+      info: multipleDuration ? `${multipleDuration} Mins` : `${eventType.length} Mins`, // TODO: Get this from props
```

Comment:
i just noticed we need to add i18n translations for both multiple and normal events